docs/visualize/most-frequent.asciidoc

```diff
@@ -1,57 +1,54 @@
 [[most-frequent]]
-== Most frequently used visualizations
+== Classic visualizations
 
-The most frequently used visualizations allow you to plot aggregated data from a <<save-open-search, saved search>> or <<index-patterns, index pattern>>.
+Visualize includes several classic visualizations that allow you to plot, visualize, and analyze your aggregated Elasticsearch data.
 
-The most frequently used visualizations include:
+Classic visualizations include:
 
-* Line, area, and bar charts
-* Pie chart
+* Line, area, bar, and pie charts
```

Comment:
Would it be useful to include a 1-line description about what each of these are useful for?